conf/webpack/production.js

```diff
@@ -0,0 +1,32 @@
+const { merge } = require('webpack-merge');
+const common = require("./common.js");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = merge(common, {
+  mode: 'production',
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: 'stylesheets/[name].css',
+      chunkFilename: '[id].css'
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(sa|sc|c)ss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
```

Comment:
I faced an issue with this on production, so please take a look at [this PR](https://github.com/hoangmirs/go-scraper/pull/30) to avoid my issue

Reply:
@hoangmirs Changed https://github.com/Lahphim/go-crawler-challenge/pull/33/commits/0dbce57a6f067582175a967d85ccab61a6abc5f9